pkg/cliplugins/claims/cmd/cmd.go

```diff
@@ -35,6 +35,12 @@ var (
 
 	# List permission claims and their respective status for all APIBindings in current workspace.
 	%[1]s claims get apibinding
+
+	# Edit the permission claims' status related to a specific APIBinding with an interactive prompt.
+	%[1]s claims get apibinding cert-manager
+
+	# Edit the permission claims' status for all APIBindings in current workspace with an interactive prompt.
+	%[1]s claims get apibinding
```

Comment:
```suggestion
	%[1]s claims edit apibinding
```

Reply:
This reads weird - maybe require `--all` or something?

Reply:
The idea was to just to list all the open claims for all bindings in case a specific binding wasn't given. It follows a similar format to the `get` command. If we add `--all` flag then we should probably do so for `get` too.